modules/utxo-lib/src/bitgo/wallet/chains.ts

```diff
@@ -102,3 +102,11 @@ export function isExternalChainCode(v: ChainCode): boolean {
 export function isInternalChainCode(v: ChainCode): boolean {
   return toChainPair(v).indexOf(v) === 1;
 }
+
+/**
+ * @return true iff chain code is a segwit address
+ */
+export function isSegwit(v: ChainCode): boolean {
+  const segwitCodes: ChainCode[] = [...chainCodesP2shP2wsh, ...chainCodesP2wsh, ...chainCodesP2tr];
```

Comment:
It might make sense to include the non-script `p2wpkh` type as well here just for completeness sake.

Reply:
Never mind I realized this is using our bitgo chaincodes and we don't have one for p2wpkh since we don't use it, so disregard.